lib/ansible/modules/cloud/huawei/hwc_network_vpc.py

```diff
@@ -34,6 +34,27 @@
         type: str
         choices: ['present', 'absent']
         default: 'present'
+    timeouts:
+        version_added: '2.9'
```

Comment:
The order should always be description, type, required, choices, default, added_in_version.

Reply:
Done